art/api/views.py

```diff
@@ -1,3 +1,27 @@
-from django.shortcuts import render
+from rest_framework.generics import ListCreateAPIView, RetrieveUpdateDestroyAPIView
+from rest_framework.response import Response
+from rest_framework import status
+from .models import Asset, Checkin
+from art.serializers import AssetSerializer, ChekinSerializer
+
+class AssetCreateApiView(ListCreateAPIView):
+    """Creates, lists item using user_id """
+    
+    serializer_class = AssetSerializer
+    # To Add code with Josiah
+
+class AssetGetUpdateDelete(RetrieveUpdateDestroyAPIView):
+    """ This gets, updates, deletes an asset using  the userid"""
+    serializer_class = AssetSerializer
+    # To Add code with Josiah
+
+class ChekinCreateApiView(ListCreateAPIView):
+    """This creates, lists chekin entries"""
+    serializer_class = ChekinSerializer
+    # To Add code with Josiah
+
+class ChekinGetUpdateDelete(RetrieveUpdateDestroyAPIView):
+    """This gets, updates, deletes a chekin entry using the userid"""
+    serializer_class = ChekinSerializer
+    # To Add code with Josiah
 
```

Comment:
Does this mean we will have to wait for the PR to be merged to proceed with the code for the views?

Reply:
yeeeepppp!!!!